test/Dialect/Torch/canonicalize.mlir

```diff
@@ -1267,3 +1267,37 @@ func.func @torch.aten.Bool.int$fold_cst() -> !torch.bool {
   %1 = torch.aten.Bool.int %int : !torch.int -> !torch.bool
   return %1 : !torch.bool
 }
+
+// CHECK-LABEL:   func.func @torch.aten.add.Tensor$canonicalize_0d() -> !torch.vtensor<[],si64> {
+// CHECK:      %[[INT6:.*]] = torch.constant.int 6
+// CHECK:      %[[INT2:.*]] = torch.constant.int 2
+// CHECK:      %[[INT0:.*]] = torch.constant.int 0
+// CHECK:      %[[PR1:.*]] = torch.prim.NumToTensor.Scalar %[[INT0]] : !torch.int -> !torch.vtensor<[],si64>
+// CHECK:      %[[PR2:.*]] = torch.prim.NumToTensor.Scalar %[[INT2]] : !torch.int -> !torch.vtensor<[],si64>
+// CHECK:      %[[PR3:.*]] = torch.prim.NumToTensor.Scalar %[[INT6]] : !torch.int -> !torch.vtensor<[],si64>
+// CHECK:      return %[[PR3]] : !torch.vtensor<[],si64>
+func.func @torch.aten.add.Tensor$canonicalize_0d() -> !torch.vtensor<[],si64> {
+    %int0 = torch.constant.int 0
+    %int2 = torch.constant.int 2
+    %int3 = torch.constant.int 3
+    %0 = torch.prim.NumToTensor.Scalar %int0 : !torch.int -> !torch.vtensor<[],si64>
+    %1 = torch.prim.NumToTensor.Scalar %int2 : !torch.int -> !torch.vtensor<[],si64>
+    %2 = torch.aten.add.Tensor %0, %1, %int3 : !torch.vtensor<[],si64>, !torch.vtensor<[],si64>, !torch.int -> !torch.vtensor<[],si64>
+    return %2 : !torch.vtensor<[],si64>
+}
+
+// CHECK-LABEL:   func.func @torch.aten.add_1.Tensor$canonicalize_0d() -> !torch.vtensor<[],si64> {
+// CHECK:      %[[INT6:.*]] = torch.constant.int 6
+// CHECK:      %[[INT2:.*]] = torch.constant.int 2
+// CHECK:      %[[PR1:.*]] = torch.prim.NumToTensor.Scalar %[[INT2]] : !torch.int -> !torch.vtensor<[],si64>
+// CHECK:      %[[PR2:.*]] = torch.prim.NumToTensor.Scalar %[[INT6]] : !torch.int -> !torch.vtensor<[],si64>
+// CHECK:      return %[[PR2]] : !torch.vtensor<[],si64>
+func.func @torch.aten.add_1.Tensor$canonicalize_0d() -> !torch.vtensor<[],si64> {
```

Comment:
nit: can you name this function `@torch.aten.add.Tensor$canonicalize_0d_vtensor_literal` (note: i've removed the `_1` after `add`)

Reply:
I see, there is another function for testing as `@torch.aten.add.Tensor$canonicalize_0d_vtensor_literal`. That is the reason I added _1.  Do you want me name them as `@torch.aten.add_1.Tensor$canonicalize_0d_vtensor_literal` and `@torch.aten.add_2.Tensor$canonicalize_0d_vtensor_literal`

Reply:
@ramiro050 I changed their names please take a look